src/kontrol/__main__.py

```diff
@@ -748,21 +462,21 @@ def parse(s: str) -> list[T]:
     build.add_argument(
         '--regen',
         dest='regen',
-        default=False,
+        default=None,
```

Comment:
These can just be removed.

Reply:
I thought so in a previous PR, but I guess omitting `default` does not always make the default `None`. At least in the case of `bool` it makes it false, although that may be caused by the `action='store_true'`, and this will overwrite the default specified in `Options.default`.